### ISSUE TITLE
Can't use SCENE SEL

### ISSUE THREAD

**Issue by mrumley02** [ldrolez/clyphx-live10#4]
I can't manage to use `[] SCENE SEL 6` to move the cursor position to scene 6.
Works in my copy of ClyphX Pro in Live 10 but not in Live 11 with this version. 
f it's limited to ClyphX Pro do you have a suggested workaround to select a scene with the cursor ready for triggering?

**Comment by ldrolez** [comment]
You only want to move the cursor to 6?
I have tried 'SCENE 6' to start scene 6 and this works.

**Comment by mrumley02** [comment]
Yes I only want to move the cursor because I am also stopping playback ready to restart on cue. Using this in the theatre.

**Comment by mrumley02** [comment]
Sorry I think I was tad ambiguous. This is a bit of an outline that I hope helps

(scenes triggering sequentially via follow actions)
`[SCENE 4] SCENE SEL 6` (move cursor to scene 6)
`[SCENE 5] SET STOP` (stop audio)
`[SCENE 6] BPM 70 ; SIG 2/4` (trigger more audio when I press enter)

The idea is that the music will stop when we hit Scene 5, wait for stage action, then hit enter to trigger Scene 6 when ready.

**Comment by ldrolez** [comment]
Thanks I see now. With Live 11 scene follow actions you could tell Live to move to scene 6 after X bars, but it would start the scene 6.

**Comment by mrumley02** [comment]
When using Scene follow actions to move through the scenes the *select next scene on launch* option only works on the scene's you trigger manually. For example, I trigger Scene 1, it selects Scene 2, Ableton plays Scenes 1-4. Scene 5 stops playback. The cursor is left on Scene 2. Normally I would use either `SEL 6` or `MST/SEL 6` to move the cursor but this doesn't seem to work anymore. Using `2/SEL 6` works fine (selecting scene 6, track 2) but I need the Master track selected so I can trigger it with the enter key.
